src/primitives/transaction.h

```diff
@@ -382,6 +380,9 @@ class CTxIn
     CScript scriptSig;
     uint32_t nSequence;
     CAssetIssuance assetIssuance;
+    /* If this is set to true, the input is interpreted as a
+     * peg-in claim and processed as such */
+    bool m_is_pegin = false;
```

Comment:
nit: this isn't bitcoin style (`fHas` vs `m_is_`).

Reply:
style guide has changed somewhat recently: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md

Reply:
Huh. I wonder why. That's an abrupt change from basically ~all of the Satoshi codebase.